synfig-studio/src/gui/app.cpp

```diff
@@ -359,7 +359,10 @@ class GlobalUIInterface : public synfigapp::UIInterface
 		dialog.set_default_response(dflt);
 
 		dialog.show_all();
-		return (Response) dialog.run();
+		int response = dialog.run();
+		if (response != Gtk::RESPONSE_OK)
```

Comment:
I think this will make it clearer what happens: 

```
		if (response == Gtk::RESPONSE_OK)
			return RESPONSE_OK;
		return RESPONSE_CANCEL;
```

Reply:
I'd prefer to keep consistency with all other cases where I check if it is not favorable (not OK) first. :)
And, indeed, I don't know why I wrote `return Response(response)` as `response` is obviously `RESPONSE_OK`.

Is this ok?
```c++
		if (response != Gtk::RESPONSE_OK)
			return RESPONSE_CANCEL;
		return RESPONSE_OK;
```
or should I write as you proposed?

Reply:
> Is this ok?
Sure.

Reply:
Done